docs/apache-airflow/start/local.rst

```diff
@@ -48,6 +48,8 @@ constraint files to enable reproducible installation, so using ``pip`` and const
     AIRFLOW_VERSION=|version|
     PYTHON_VERSION="$(python --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
     # For example: 3.6
+    # If you are using Python3 
+    # PYTHON_VERSION="$(python3 --version | cut -d " " -f 2 | cut -d "." -f 1-2)"
```

Comment:
We only support Python3 on the`main` branch.

Reply:
Python 2 is 'gone' more and more distribution removes it as being 'default". You should only run Airflow in Python3 (virtual) environment  where python is python 3. Actually if your default python is python 2, airflow will likely not work well.

You can even install `python-is-python3` package in older versions of distributions https://askubuntu.com/questions/320996/how-to-make-python-program-command-execute-python-3 

We should promote using python2 removal once and for all rather than try to workaround it and using `python' as python3 equivalent is a very good way to do it. Let's keep it the way it is.